app/login/index.html

```diff
@@ -16,27 +16,38 @@
         </p>
       </div>
     </noscript>
-    <h1>Login</h1>
-    <p>Login to your Mars Rover account to access your photos and data</p>
-    <form class="login" method="POST" action="/authenticate">
-      <label for="username">Username</label>
-      <input
-        type="text"
-        placeholder="Enter Username"
-        id="username"
-        name="username"
-        required
-      />
-      <label for="password">Password</label>
-      <input
-        type="password"
-        placeholder="Enter Password"
-        id="password"
-        name="password"
-        required
-      />
-      <button type="submit" value="submit">Login</button>
-      <a href="/register">Register an account</a>
-    </form>
+
+    <main id="login-page">
+      <section id="login-container">
+        <h1>Login</h1>
+        <p>Login to your Mars Rover account to access your photos and data</p>
+        <form class="login" method="POST" action="/authenticate">
+          <label for="username">Username</label>
+          <input
+            type="text"
+            placeholder="Enter Username"
+            id="username"
+            name="username"
+            required
+          />
+          <br />
```

Comment:
As you've said, let's concentrate on the register stuff, and I suspect you already know, but we should avoid using <br/> for spacing

Reply:
Yes :) Planning to refactor this 👍

Reply:
Thanks for highlighting these, I forgot to remove them